### DISCUSSION TITLE
Unable to save Rest API Connectore settings

### DISCUSSION THREAD

**Discussion by jkatsabanissp** [Budibase/budibase#8498] [Help] [answered]
I am trying to add a REST API connector in v2.1.8 but no matter what I do, the settings for always revert to the default. For example, if i select "Reject Unauthorized" or try to add an Auth type it will never save the settings.
![Screenshot 2022-11-02 165301](https://user-images.githubusercontent.com/88497085/199600781-2d2e4bdf-ee89-4058-9012-c967dd9b10dc.png)

**Comment by jkatsabanissp** [comment]
Furthermore, the "Save button never toggles no matter what modifications I make

**Reply by jedman14** [reply]
I seem to have this same problem, I cannot add any Authentication on this page.

**Comment by jkatsabanissp** [comment, marked as answer]
So it turns out updating and upgrading packages in linux fixed this problem for me. Performed the following steps in console:

sudo apt-get update
sudo apt-get upgrade
